framework/tests/returnsreceiverlombok/BuilderMethodRef.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Optional;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.checkerframework.common.returnsreceiver.qual.*;
+
+@Builder
+@Accessors(fluent = true)
+public class BuilderMethodRef {
+    @Getter @Setter @lombok.NonNull String foo;
+    @Getter @Setter Object bar;
+
+    public static void test(Optional<Object> opt) {
+        BuilderMethodRefBuilder b = builder().foo("Hello");
+        opt.ifPresent(b::bar);
+        b.build();
+    }
+}
+
+class CustomBuilderMethodRefBuilder extends BuilderMethodRef.BuilderMethodRefBuilder {
```

Comment:
@kelloggm this subclass has the wrapper methods that ensure that `@This` annotations are getting inserted correctly for Lombok.  I think this should be sufficient for testing, but let us know if you think they belong elsewhere or need better documentation

Reply:
These are great. My comment wasn't clear, but I was referring to `framework/tests/returnsreceiverlombok/BuilderTest.java`, which doesn't test anything other than that the RR checker issues no error on that file. It should check that the setters have `@This` annotations, too.

Reply:
Now added

Reply:
BTW, these extra tests are how we discovered the need for the extra flag for Lombok.  So they are definitely useful 👍